### DISCUSSION TITLE
OpenDTU von Tremex verliert die Verbindung

### DISCUSSION THREAD

**Discussion by HeinzEagle** [tbnobody/OpenDTU#2044] [Q&A]
Hallo an alle,
ich verzweifle so langsam. Habe mir eine DTU von Tremex für einen Wechselrichter von Hoymiles HMS-1600 bestellt.
Nachdem er angekommen war habe ich die notwendigen Daten eingegeben und innerhalb einiger Sekunden war die Verbindung zum WR aufgebaut und die Daten kamen rein. Soweit sogut, am nächsten Morgen hatte er erstmals keine Verbindung was nach ca. 2 Stunden dann aber wieder OK war. Also lief es und zeigte Daten an. Wiederrum einen Tag weiter fand er einen ganzen Tag keine Verbindung, den Tag drauf lief er wieder. Jetzt, nach einer Woche kommt gar keine Verbindung zustande. Habe die DTU reklamiert und sie wurde getauscht., Jetzt das gleiche Spiel von vorne. Est gut und im Laufe der Zeit immer schlechter. Bin mit meiner Weisheit total am Ende. 
Hat jemand einen Tipp für mich woran es liegen kann?!
Habe auch schon andere Firmware Stände geflasht, hilft aber alles nichts.

**Comment by ghost** [comment]


**Reply by HeinzEagle** [reply]
Hallo Schneeer, ganz lieben Dank für deine Antwort. Ja, das Netzteil bringt 3A, ich habe sogar ein Oszyloskop darngehalten und die Gleichspannung sieht echt sauber aus. Den Stützkondensator werde ich noch einbauen, bisher ist keiner verbaut. 
Danke für den Tipp, werde berichten.

**Reply by HeinzEagle** [reply]
So, habe mal nachgesehen wegen dem Elko und dort ist ein 100 µF verbaut. Das wird dann auch wohl nicht der Fehler sein. Was mir jetzt aufgefallen ist, ich habe die Firmware V24.1.26 installiert. Ich benötige diese Version weil ansonsten das Script (Pyton) auf meinem Raspberry die Daten der höheren Versionen nicht verarbeiten kann. Wenn ich keine Verbindung zu den WR bekomme und die Firmware über die Oberfläche "Firmware-Aktualisierung" neu Aufspiele also die den gleiche Release Stand dann ist die DTU innerhalb weniger Sekunden wieder verbunden. Es macht den Eindruck als würde irgendein Cash vollaufen der durch das Flashen wieder glöscht wird und sie kann wieder Daten verarbeiten. Nur da stoße ich an meine Grenzen....

**Reply by HeinzEagle** [reply]
Man liest immer wieder von Kondensatoren einlöten, andere Netzteile…..
Ja, Kondensatoren waren bei mir auf der Platine und die Netzteil habe ich auch gewechselt. Nur war das eine Steckernetzteil genauso beschissen wie 4 andere auch. Nun habe ich eines genommen was deutlich teurer war mit 4 usb Ports, hat über 20€ gekostet. Seit dem das Netzteil meine DTU versorgt ist beim Sonnenaufgang meine DTU wach bis es wieder dunkel wird. Das seit 4 Tagen, hoffe es bleibt so. Wenn ja ist es tatsächlich ein Spannungsproblem gewesen. Ich hoffe das ich hiermit ein wenighelfen konnte.
